nncf/quantization/algorithms/min_max/torch_backend.py

```diff
@@ -188,6 +183,13 @@ def get_statistic_collector(
             collector.register_statistic_branch(container_key, reducer, aggregator)
         return collector
 
+    @staticmethod
+    def transform_to_inference_graph(graph: NNCFGraph) -> NNCFGraph:
```

Comment:
It looks like the right place of `transform_to_inference_graph` into `passes.py`

Reply:
We should decide where is the best place for that. @AlexanderDokuchaev suggest algorithm backend is the best way for this method

Reply:
I reverted `transform_to_inference_graph` function back